src/main/scripts/validate_deployment.groovy

```diff
@@ -59,7 +74,40 @@ println('command:')
 println(command.join(' '))
 println('')
 
-def process = command.execute()
+def exitCode = 0
```

Comment:
Will Do. I by no means consider myself a developer, so I am still learning pretty much everything. I had just used what was used to create the process in the import_objects.groovy from this plugin. For future reference, I will take a look at the CommandHelper and make some changes to this at a later date once I am more comfortable writing my own stuff.  Thanks!